video2dataset/video2dataset.py

```diff
@@ -7,51 +7,66 @@
 
 
 def video2dataset(
-  src,
-  dest="",
-  output_format="webdataset",
-  metadata_columns="",
+    src,
+    dest="",
+    output_format="webdataset",
+    metadata_columns="",
+    get_audio=False,
```

Comment:
can we change this to the "video_formats" parameter we discussed? This would be a comma separated string like "mp4" or "mp4,mp3" and if you want audio you can just check for "mp3" in video_formats

Reply:
Already implemented, working on testing